api/src/main/java/org/pmiops/workbench/api/UserMetricsController.java

```diff
@@ -180,7 +181,7 @@ public ResponseEntity<WorkspaceResourceResponse> getUserRecentResources() {
     final ImmutableList<DbUserRecentlyModifiedResource> workspaceFilteredResources =
         userRecentlyModifiedResourceList.stream()
             .filter(r -> idToFirecloudWorkspace.containsKey(r.getWorkspaceId()))
-            .filter(this::hasValidBlobIdIfNotebookNamePresent)
```

Comment:
this is the filtering method - I have renamed it so it now applies to other resources